util/cert/cert.go

```diff
@@ -130,7 +131,11 @@ func ParseTLSCertificatesFromPath(sourceFile string) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer fileHandle.Close()
+	defer func() {
+		if err = fileHandle.Close(); err != nil {
+			log.Errorf("error closing file %q: %v", fileHandle.Name(), err)
+		}
```

Comment:
@my-git9 your work inspired some additional logging changes: https://github.com/argoproj/argo-cd/pull/10256/files#diff-25220a29fb80773c52f3ff40158cbfbe138517c2d08b8a204c7091af468b82aeR171

Would you mind adding this to the log statement?

```
.WithFields(log.Fields{
				common.SecurityField:    common.SecurityMedium,
				common.SecurityCWEField: 775,
			})
```

That'll just make it easier for folks to monitor these issues.

Reply:
Sure, I will do it later

Reply:
It has been updated. The previous PR was also changed like this, like this: https://github.com/argoproj/argo-cd/pull/10127/files, do I need to submit another PR to adjust it?

Reply:
Only if you want to! Adding security logging is helpful, but optional.  :-)

Reply:
OK, I can try it